### PR TITLE
MWPW-139915: Delivery of Spectrum Web Components (#1663)

### PR DESCRIPTION
**Pull request by yesil**
* MWPW-139915: Spectrum Web Components delivery for commerce

This PR brings a new `libs/deps/lit-all.min.js` version.

Resolves: [MWPW-139915](https://jira.corp.adobe.com/browse/MWPW-139915)

**Test URLs:**
- Before: https://main--milo--adobecom.hlx.page/docs/library/kitchen-sink/merch-card?martech=off&georouting=off
- After: https://mwpw-139915-main--milo--yesil.hlx.page/docs/library/kitchen-sink/merch-card?martech=off&georouting=off

### CODE REVIEW

**Review by honstar — APPROVED** [review]

File: libs/features/spectrum-web-components/build.mjs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+const LIT_PATH_PATTERN = /^lit(\/.*)?$/;
+const ICON_PATH_PATTERN = /^@spectrum-web-components\/icon\/src\/spectrum-icon-(.*).css.js/;
+
+const IGNORE_PATHS = [
+  '@spectrum-web-components/modal/src/modal-wrapper.css.js',
+  '@spectrum-web-components/modal/src/modal.css.js',
+];
+
+const target = 'es2021';
```
Comment (npeltier) [inline libs/features/spectrum-web-components/build.mjs:21]: upper case please

File: libs/features/spectrum-web-components/build.mjs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+        if (LIT_PATH_PATTERN.test(args.path)) {
+          return {
+            path: '/libs/deps/lit-all.min.js',
+            external: true,
+          };
+        }
+        if (ICON_PATH_PATTERN.test(args.path)) {
+          const iconName = args.path.match(ICON_PATH_PATTERN)[1];
+          return {
+            path: `/libs/features/spectrum-web-components/dist/icons/${iconName}.js`,
```
Comment (npeltier) [inline libs/features/spectrum-web-components/build.mjs:46]: may be this template should be a const next to ICON_PATH_PATTERN l14?

File: libs/features/spectrum-web-components/build.mjs [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+            external: true,
+          };
+        }
+
+        if (/@spectrum-web-components/.test(args.path) && args.path.indexOf(`@spectrum-web-components/${entry}`) < 0 && !IGNORE_PATHS.includes(args.path)) {
+          // get the first folder after @spectrum-web-components
+          const [, module] = args.path.split('/');
+          return {
+            path: `/libs/features/spectrum-web-components/dist/${module}.js`,
```
Comment (npeltier) [inline libs/features/spectrum-web-components/build.mjs:55]: same here

**Review by npeltier — APPROVED** [review]

File: libs/features/spectrum-web-components/README.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
+# Spectrum Web Components
+
+This repository provides individual Spectrum Web Components for consumption in SPA like pages.<br>
```
[the remark this answers is not in the document]
Comment (Axelcureno) [reply libs/features/spectrum-web-components/README.md:3]: or `SPA-like pages` maybe?

**Review by Axelcureno — APPROVED** [review]

File: libs/features/spectrum-web-components/README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (yesil) [reply libs/features/spectrum-web-components/README.md:3]: yes

**Pull request merged**